cassdegrees/static/css/style.css

```diff
@@ -27,3 +27,11 @@
     justify-content: center;
     margin: 12px;
 }
+
+select.tfull {
+    width: 70.25%;
+}
+
+select.text{
+    border-radius: 3px;
```

Comment:
Not specific to this PR, but:

http://webpublishing.anu.edu.au/web-style-guide/style-sampler.php
`Please avoid using inline styles or creating custom style sheets`

We should be justifying each use of rules in this file.

Reply:
Not sure if it was mentioned above but it didn't seem like there was a `<select> `version of `tfull` in their css file hence the creation of new rules.